libr/util/str.c

```diff
@@ -3564,6 +3565,26 @@ R_API const char *r_str_str_xy(const char *s, const char *word, const char *prev
 	return d;
 }
 
+R_API size_t r_charset_encode_str(ut8 *asciistr, ut8 *in, size_t len_input, RCharset *r_char, size_t len_charset) {
+	bool is_in_set;
+	size_t i, j;
+	for (i = 0; i < len_input; i ++) {
+		is_in_set = false;
+		for (j = 0; j < len_charset; j ++) {
+			//if we want an hashtable, let's do: asciistr[i] = custom_charset[j].ch; We are currently debating.
+			if (in[i] == r_char->custom_charset[j].hx) {
+				asciistr[i] = r_char->custom_charset[j].ch;
+				is_in_set = true;
+			}
+		}
+		if (is_in_set == false) {
```

Comment:
use `if (!is_in_set) {`

Reply:
@trufae true again I should avoid c++ pratices!